plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -95,6 +96,20 @@
 public class ParquetPageSourceFactory
         implements HivePageSourceFactory
 {
+    /**
+     * If this column is passed as one of the columns for {@code createPageSource},
+     * it will be populated as an additional column containing the index of each
+     * row read.
+     */
+    public static final HiveColumnHandle ROW_INDEX_COLUMN = new HiveColumnHandle(
```

Comment:
I think these usually go in `HiveColumnHandle`

Reply:
yeah, keep it here. We can even call it PARQUET_ROW_INDEX_COLUMN.

Reply:
Renamed to `PARQUET_ROW_INDEX`.